shared/oae/css/oae.components.css

```diff
@@ -1009,7 +1014,6 @@ div.oae-thumbnail i.fa {
 .oae-tile,
 .oae-list.oae-list-grid > li.oae-list-actions {
     position: relative;
-    overflow: hidden;
```

Comment:
I don't know why it was there, but I'm seeing borders being chopped off in the activity tile depending on the width of the screen because of this. I think they are one-off pixel issues in at least chrome due to %-based widths.

I can't think of any situation where we actually want to chop something out of the UI with `overflow: hidden`, can you? If we are, we *probably* want to do something else, so I was hoping it would come up in testing, but it hasn't so far.